### PR TITLE
ceph-nvmeof baremetal suite

### PR DESCRIPTION
**Pull request by sunilkumarn417**
- Add baremetal support for ceph-nvmeof.

### CODE REVIEW

File: tests/nvmeof/test_ceph_nvmeof_gateway.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
 LOG = Log(__name__)
 
+# def run_cli_with_check(gw, *args, **kwargs):
```
Comment (rahullepakshi) [inline tests/nvmeof/test_ceph_nvmeof_gateway.py:19]: I believe this comment is here by mistake. We could remove as we have not implemented this

**Comment by rahullepakshi** [comment]
Do we have test run logs for this implementation?

**Review by rahullepakshi — APPROVED** [review]
LGTM

**Review by Manohar-Murthy — APPROVED** [review]
lgtm.

**Pull request merged**